src/core/service.c

```diff
@@ -3473,7 +3473,17 @@ static void service_sigchld_event(Unit *u, pid_t pid, int code, int status) {
                         switch (s->state) {
 
                         case SERVICE_START_POST:
+                                _fallthrough_;
```

Comment:
this shouldn't be necessary? the compiler shouldn't complain about stacked cases iirc

Reply:
You are right, sorry for the noise.  Pushed changes.